documentation/my-homeworks/week3.2/src/components/app.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import CreateTodo from './create-todo';
```

Comment:
It's better to name *Components* as names not verbs, like `TodoForm` not `CreateTodo`